daemon-systemd.in

```diff
@@ -10,7 +10,7 @@ PIDFile=@lockfile@
 PermissionsStartOnly=true
 ExecStartPre=-/usr/bin/mkdir /var/run/naemon
 ExecStartPre=/usr/bin/chown -R naemon:naemon /var/run/naemon/
-ExecStartPre=@bindir@/naemon -vp @pkgconfdir@/naemon.cfg
+ExecStartPre=su - naemon -c "@bindir@/naemon -vp @pkgconfdir@/naemon.cfg"
```

Comment:
Everything except the /var/run/naemon is in fact already created by the package scripts. But /var/run/naemon must be created prior start and could be removed. So yes, the `PermissionsStartOnly` is there because of the `mkdir`. Newer systemds have a `RuntimeDir` which solves that, but most of the systems we build naemon for are too old for that option.